docs/source/yelpsoa_configs.rst

```diff
@@ -281,12 +281,23 @@ instance MAY have:
     scripts will send a CRITICAL alert.
 
 In addition, each instancename MAY configure additional Marathon healthcheck
-options:
+options (Read the official
+`mesos documentation <https://mesos.readthedocs.io/en/latest/health-checks/>`_
+for more low-level details:
 
-  * ``healthcheck_mode``: One of ``cmd``, ``tcp``, ``http``, or ``https``. If your
-    service uses Smartstack, then this must match the value of the ``mode`` key
-    defined for this instance in ``smartstack.yaml``. If set to ``cmd`` then
-    PaaSTA will execute ``healthcheck_cmd`` and examine the return code.
+  * ``healthcheck_mode``: One of ``cmd``, ``tcp``, ``http``, or ``https``.
+    If set to ``http`` or ``https``, a ``curl`` command will be executed
+    inside the container. ``curl`` must be installed in the container for
```

Comment:
I don't think this is true: you need curl on the host, and it mesos executes the cmd inside the network namespace of the container.

Reply:
Huh. I thought for sure this was the case, but I can't find any evidence of it anymore.